SeleniumSolarTechnology/SolarTechnology/Pages/CartPage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SolarTechnology.Pages
+{
+    internal class CartPage: BasePage
+    {
+        private By loader = By.Id("p_prldr");
```

Comment:
private By loader and WaitForLoader() are present in each Page class, so we should move them to BasePage

Reply:
Thanks, I guessed that too..;). Done.